### DISCUSSION TITLE
GOMA video tutorial

### DISCUSSION THREAD

**Discussion by 2sekarraja** [goma/goma#395] [General]
Hello everyone,
Recently i installed GOMA on my machine after looking through its capabilities. Is it possible to have a basic video tutorial for goma problems on any topic? if i get one it will be helpful for my research

**Comment by wortiz** [comment]
I don't know that we have the resources to create a video tutorial easily for free.

We do have a beginners tutorial: https://www.gomafem.com/documentation.html#goma-beginners-tutorial which offers an introduction to the running goma and what each file does, as well as a general overview.

We do require [ExodusII](https://sandialabs.github.io/seacas-docs/sphinx/html/index.html#exodus-library) meshes either from Cubit (https://cubit.sandia.gov/ or https://coreform.com/products/coreform-cubit/ coreform offers a free education version with limited mesh) or from some mesh conversion/creation tool ( for example https://github.com/wortiz/msh2exo).

If you tell us a little more about what you are trying to model we can probably point you in the right direction of available functionality in Goma
